llvm/test/CodeGen/SPIRV/debug-info/basic-global-di.ll

```diff
@@ -0,0 +1,129 @@
+; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+
```

Comment:
Do we generate valid output SPIR-V in the case of this test? If so, we could have a `spirv-val` line added as well. Otherwise, you could add a `TODO: spirv-val`. We have similar lines in other LITs.

Reply:
Done.